examples/rpc/partner-plugin/services/data/v2/shared_data.go

```diff
@@ -0,0 +1,76 @@
+package helpers
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"path/filepath"
+	"sync"
+
+	accommodationv2 "buf.build/gen/go/chain4travel/camino-messenger-protocol/protocolbuffers/go/cmp/services/accommodation/v2"
+)
+
+var (
+	properties []accommodationv2.PropertyExtendedInfo
+	loadOnce   sync.Once
+)
+
+func LoadPropertiesMockData() []accommodationv2.PropertyExtendedInfo {
+	loadOnce.Do(func() {
+		// Assuming the JSON file is relative to the current file
+		currentDir, err := os.Getwd()
+		if err != nil {
+			log.Printf("Error getting current directory: %v", err)
+			return
+		}
+
+		filePath := filepath.Join(currentDir, "../../examples", "rpc", "partner-plugin", "mock_data", "properties.json")
+		data, err := os.ReadFile(filePath)
+		if err != nil {
+			log.Printf("Error reading properties file: %v", err)
+			return
+		}
+
+		if err := json.Unmarshal(data, &properties); err != nil {
+			log.Printf("Error unmarshaling properties: %v", err)
+			return
+		}
+
+		log.Printf("Successfully loaded %d properties", len(properties))
+	})
+
+	return properties
+}
+
+// ReloadPropertiesMockData reloads the properties data from the JSON file
+func ReloadPropertiesMockData() error {
```

Comment:
Why is reloading of static mock data required?
And this looks like >90% code duplication of the load function.

Reply:
this part has been removed with the refactoring of loading the json file and is no longer needed